python/py-graveyard/Portfile

```diff
@@ -473,6 +473,7 @@ py-webob                1.8.2_1     26
 py-werkzeug             0.14.1_1    26 33
 py-wordaxe              0.3.3_1     26
 py-w3lib                1.9.0_1     33
+py-xarray               0.11.0      34
```

Comment:
It's better to use `0.10.3_1` here. If a user is prompted to upgrade `py34-xarray` from 0.10.3 to 0.11.0, he/she may expect an upstream update, while it's actually not.

Reply:
Ok, done!